iothub_client/samples/pnp/pnp_temperature_controller/pnp_temperature_controller.c

```diff
@@ -148,65 +144,87 @@ static void SetEmptyCommandResponse(unsigned char** response, size_t* responseSi
 }
 
 //
-// PnP_TempControlComponent_DeviceMethodCallback is invoked by IoT SDK when a device method arrives.
+// PnP_TempControlComponent_InvokeRebootCommand processes the reboot command on the root component.
+//
+static int PnP_TempControlComponent_InvokeRebootCommand(JSON_Value* rootValue)
+{
+    int result;
+
+    if (json_value_get_type(rootValue) != JSONNumber)
+    {
+        LogError("Delay payload is not a number");
+        result = PNP_STATUS_BAD_FORMAT;
+    }
+    else
+    {
+        // See caveats section in ../readme.md; we don't actually respect the delay value to keep the sample simple.
+        int delayInSeconds = (int)json_value_get_number(rootValue);
+        LogInfo("Temperature controller 'reboot' command invoked with delay %d seconds", delayInSeconds);
+        result = PNP_STATUS_SUCCESS;
+    }
+    
+    return result;
+}
+
+//
+// PnP_TempControlComponent_CommandCallback is invoked by IoT SDK when a command arrives.
 //
-static int PnP_TempControlComponent_DeviceMethodCallback(const char* methodName, const unsigned char* payload, size_t size, unsigned char** response, size_t* responseSize, void* userContextCallback)
+static int PnP_TempControlComponent_CommandCallback(const char* componentName, const char* commandName, const unsigned char* payload, size_t size, const char* payloadContentType, unsigned char** response, size_t* responseSize, void* userContextCallback)
 {
```

Comment:
nit: need line break

Reply:
@hihigupt  - I'm assuming what you mean here is that this line runs on WAY too long and that the params should be split between lines?  I agree and am fixing.

If you mean something else - like CRLF | LF or whatever type line breaks are screwed up - please let me know.

Reply:
Pretty sure its the too long and needing an extra line.  I've fixed this as suggested, so resolving.